Jenkinsfile

```diff
@@ -97,11 +99,20 @@ pipeline {
 	post {
 	    success {
 			script {
+			    unstash 'site'
+				def siteFilesToPush = findFiles(glob: 'site/target/repository/**')
+				def mvnFileVersion = sh script: 'mvn help:evaluate -Dexpression=project.version -q -DforceStdout', returnStdout: true
+
+				for (i = 0; i < siteFilesToPush.length; i++) {
+					echo "rsync -Pzrlt --rsh=ssh --protocol=28 ${siteFilesToPush[i].path} ${UPLOAD_LOCATION}/snapshots/rsp-server/p2/${mvnFileVersion}"
+				}
+
 			    unstash 'zips'
 				def filesToPush = findFiles(glob: '**/*.zip')
 				for (i = 0; i < filesToPush.length; i++) {
 					sh "rsync -Pzrlt --rsh=ssh --protocol=28 ${filesToPush[i].path} ${UPLOAD_LOCATION}/snapshots/rsp-server/"
```

Comment:
Would be nice if the extracted version does not contain SNAPSHOT to deploy to stable

Reply:
I'd rather keep the snapshot update sites different than teh stable ones. The stable ones won't have SNAPSHOT in their version variable and so will be a different repo.

Reply:
That was my purpose if the version does not have SNAPSHOT (stable build) then deploy to stable directory instead of snapshots on d.j.o

Reply:
Right now this is the only folder that's up. Turns out we don't even have a 'stable' for our final distributions. They're all shoved in the same place.  I'll fix both of them together but it's not the first priority at this moment.